js/packages/web/src/views/art/index.tsx

```diff
@@ -118,12 +122,31 @@ export const ArtView = () => {
                       {!creator.verified &&
                         (creator.address === pubkey ? (
                           <Button
+                            loading={validating}
                             onClick={async () => {
+                              setValidating(true);
+
                               try {
-                                await sendSignMetadata(connection, wallet, id);
+                                const txid = await sendSignMetadata(connection, wallet, id);
+
+                                const tx = await connection.getTransaction(txid, {
+                                  commitment: 'confirmed',
+                                });
+
+                                const keys = tx?.transaction.message.accountKeys || [];
+
+                                const patch = await loadMultipleAccounts(
+                                  connection,
+                                  keys.map(k => k.toBase58()),
+                                  'confirmed',
+                                );
+                                
+                                patchState(patch);
```

Comment:
Using a similar and advantage technique of looking up the accounts changed from the transaction and patching the state instead of relying on subscription changes. RPC isnt great at handling a large amount of subscriptions so should be reserved for listening for transaction confirmation only.